app/models/miq_worker/runner.rb

```diff
@@ -313,7 +313,7 @@ def heartbeat_to_file(timeout = nil)
     return if skip_heartbeat?
 
     timeout ||= worker_settings[:heartbeat_timeout] || Workers::MiqDefaults.heartbeat_timeout
-    File.write(@worker.heartbeat_file, (Time.now.utc + timeout).to_s)
+    File.write(@worker.heartbeat_file, (Time.now.to_i + timeout).to_s)
```

Comment:
To `.to_s` is not necessary...Not sure why it's even there.

Reply:
done...